.gitignore

```diff
@@ -273,4 +273,7 @@ libpqxx-*.deb*
 wal.log
 
 # OLTP output
-**/oltp_output_*
\ No newline at end of file
+**/oltp_output_*
+
+# Personal github config file
```

Comment:
Is this a Jekyll artifact? Guessing based on the commit history.

Reply:
Yup, thanks, updated!